docs/prow/assets/kyma-rel-jobs.svg

```diff
@@ -0,0 +1,2 @@
 font-size: 13px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 221px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">Presubmit jobs triggered automatically</div></div></foreignObject><text x="111" y="13" fill="#000000" text-anchor="middle" font-size="13px" font-family="Helvetica">Presubmit jobs triggered automatically</text></switch></g><g transform="translate(59.5,629.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="273" height="13" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 13px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; white-space: nowrap;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">Presubmit jobs triggered by comments on a PR</div></div></foreignObject><text x="137" y="13" fill="#000000" text-anchor="middle" font-size="13px" font-family="Helvetica">Presubmit jobs triggered by comments on a PR</text></switch></g><g transform="translate(59.5,671.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="86" height="13" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 13px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; white-space: nowrap;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">Postsubmit job</div></div></foreignObject><text x="43" y="13" fill="#000000" text-anchor="middle" font-size="13px" font-family="Helvetica">Postsubmit job</text></switch></g><path d="M 162 500 L 162 10" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 162 140 L 172 140 Q 182 140 192 140 L 335.63 140" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 340.88 140 L 333.88 143.5 L 335.63 140 L 333.88 136.5 Z" fill="#959ca4" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 163 255 L 182.5 254.67 Q 192.5 254.5 194.07 254.58 L 195.64 254.67" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 200.88 254.94 L 193.71 258.07 L 195.64 254.67 L 194.08 251.08 Z" fill="#959ca4" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 622 360 L 622 70" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 622 190 L 635.63 190" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 640.88 190 L 633.88 193.5 L 635.63 190 L 633.88 186.5 Z" fill="#959ca4" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/></svg>
```

Comment:
Postsubmit job is also triggered automatically so this information should appear on diagram's legend.